### ISSUE TITLE
[millipeda] missing docstrings in examples (Trac #1248)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1242]
the scripts in resources/examples need a brief explanation what they do and what kind of input they expect.

Migrated from https://code.icecube.wisc.edu/ticket/1248
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "the scripts in resources/examples need a brief explanation what they do and what kind of input they expect.", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[millipeda] missing docstrings in examples", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2015-08-20T16:39:10", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T16:59:31: 
* jbraun changed priority from "blocker" to "minor"
* jbraun changed status from "new" to "assigned"
* jbraun changed owner from "jbraun@icecube.wisc.edu" to "jbraun"

**Comment by icecube-trac** [comment]
Trac update at 20150825T17:42:08: 
* jbraun commented:

> In changeset 136844:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136844"
> Add basic docstrings to example scripts. Closes #1242.
> ```


* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
